### PR TITLE
Improve K8S monitoring usefulness

### PR DESCRIPTION
**Pull request by vponomaryov**
This PR consists of following parts:
1) Use same scrape interval and timeout in K8S prometheus as in the SCT one
2) Make K8S grafana be accessible out-of-cluster
3) Simplify K8S monitoring health checker removing redundant parts.

## PR pre-checks (self review)

- [x] I followed [KISS principle](https://en.wikipedia.org/wiki/KISS_principle) and [best practices](https://docs.google.com/document/d/1jihgOKb5iGRlD8_HQ92O0JbLk1kASUoZT23i_MXFSKI)
- [x] I didn't leave commented-out/debugging code
- [x] I added the relevant `backport` labels
- [ ] ~~New configuration option are added and documented (in `sdcm/sct_config.py`)~~
- [ ] ~~I have added tests to cover my changes (Infrastructure only - under `unit-test/` folder)~~
- [x] All new and existing unit tests passed (CI)
- [ ] ~~I have updated the Readme/doc folder accordingly (if needed)~~

### CODE REVIEW

File: sdcm/cluster_k8s/__init__.py [hunk 1]
```diff
@@ -766,7 +766,12 @@ def deploy_monitoring_cluster(
             helm_values.set('alertmanager.alertmanagerSpec', monitoring_affinity_rules)
             helm_values.set('prometheusOperator', monitoring_affinity_rules)
             prometheus_values = {
-                'prometheusSpec': monitoring_affinity_rules,
+                'prometheusSpec': {
+                    'affinity': monitoring_affinity_rules['affinity'],
```
Comment (dkropachev) [inline sdcm/cluster_k8s/__init__.py:770]: Will fail if monitoring_affinity_rules = {}

File: sdcm/cluster_k8s/__init__.py [hunk 2]
```diff
@@ -766,16 +766,32 @@ def deploy_monitoring_cluster(
...
                     # nodeIp:30090 will redirect traffic to prometheusPod:9090
                     'type': 'NodePort',
-                    'nodePort': 30090,
+                    'nodePort': self.k8s_prometheus_external_port,
                 },
             }
             helm_values.set('prometheus', prometheus_values)
-            helm_values.set('grafana', monitoring_affinity_rules)
+            grafana_values = {
+                'affinity': monitoring_affinity_rules['affinity'],
```
Comment (dkropachev) [inline sdcm/cluster_k8s/__init__.py:784]: Will fail if monitoring_affinity_rules = {}

**Review by dkropachev — CHANGES_REQUESTED** [review]

File: sdcm/cluster_k8s/__init__.py (hunk 1 above)
Comment (vponomaryov) [reply sdcm/cluster_k8s/__init__.py]: Good catch. Fixed.

File: sdcm/cluster_k8s/__init__.py (hunk 2 above)
Comment (vponomaryov) [reply sdcm/cluster_k8s/__init__.py]: Good catch. Fixed.

**Review by dkropachev — APPROVED** [review]

**Review by bentsi — APPROVED** [review]

**Review by bentsi — APPROVED** [review]

**Pull request merged**